src/hotspot/os_cpu/linux_arm/linux_arm_32.S

```diff
@@ -23,38 +23,49 @@
 
 
         # NOTE WELL!  The _Copy functions are called directly
-	# from server-compiler-generated code via CallLeafNoFP,
-	# which means that they *must* either not use floating
-	# point or use it in the same manner as does the server
-	# compiler.
+        # from server-compiler-generated code via CallLeafNoFP,
+        # which means that they *must* either not use floating
+        # point or use it in the same manner as does the server
+        # compiler.
 
+        .globl  SpinPause
+        .hidden  SpinPause
+        .type SpinPause, %function
         .globl _Copy_conjoint_bytes
-	.type _Copy_conjoint_bytes, %function
+        .hidden _Copy_conjoint_bytes
```

Comment:
This triggered a GHA failure... apparently the method `_Copy_conjoint_bytes` was removed a long time ago, and the `.globl` / `.type` declarations are the only things left. They should also be removed.